x-pack/plugins/ml/public/application/components/data_grid/common.ts

```diff
@@ -115,14 +115,42 @@ export const getRuntimeFieldsMapping = (
       }
     });
   }
-  if (isPopulatedObject(clonedRuntimeMappings)) {
-    combinedRuntimeMappings = { ...combinedRuntimeMappings, ...clonedRuntimeMappings };
+  if (isPopulatedObject(runtimeMappings)) {
+    combinedRuntimeMappings = { ...combinedRuntimeMappings, ...runtimeMappings };
   }
   return Object.keys(combinedRuntimeMappings).length > 0
     ? { runtime_mappings: combinedRuntimeMappings }
     : {};
 };
 
+export function getCombinedRuntimeMappings(
```

Comment:
If I add an extra `double` runtime field for a numeric field using the JSON editor, the histogram chart isn't displayed in the grid:

![image](https://user-images.githubusercontent.com/7405507/113285723-31764c80-92e3-11eb-89f8-d2f25ee890da.png)

If I repeat this workflow in transforms, it does display the mini histogram. The histogram is also displayed in the DFA wizard if the field is defined in the index pattern.

Reply:
There is also an issue with editing the definition of an existing runtime field defined in the index pattern. After applying the changes, the table cell values are updated as expected, but the histograms still plot using the previous mapping. Here I update the multiplier from `10` to `10000`:

![image](https://user-images.githubusercontent.com/7405507/113287481-8dda6b80-92e5-11eb-946e-c59d978c097b.png)

Reply:
Both of these issues are fixed in c224c8cf620e4613ef0a39ded34950f348bb9426
Just needed to send the updated runtime fields to the histogram fetching functionality. Thanks for catching this! 🙏